### ISSUE TITLE
native-run stuck

### ISSUE THREAD

**Issue by Incubbus** [ionic-team/native-run#81]
**Version:**
0.2.8

**Description:**

After running `ionic cordova run ios -l --address=0.0.0.0`
when the process reaches native-run, nothing happens.

Even after 15 minutes, nothing happens on the command line or on the phone.

**Command Output:**

When running the command with the verbose flag it gets to

`[native-run] 2019-10-16T15:54:26.375Z protocol:usbmux Response: {"PairRecordData":{"type":"Buffer","data":[60,63,120,109,108,32,118,10...`
and is stuck there...

**Environment Info:**

```
Ionic:

   Ionic CLI          : 5.4.4 (/usr/local/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.9
   @ionic/app-scripts : 3.2.4

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.0.0, ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-webview 1.2.1, (and 19 other plugins)

Utility:

   cordova-res : 0.6.0 (update available: 0.8.0)
   native-run  : 0.2.8 (update available: 0.2.9)

System:

   Android SDK Tools : 26.1.1 (/Users/xxx/Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 8.0.2
   NodeJS            : v12.6.0 (/usr/local/Cellar/node/12.6.0/bin/node)
   npm               : 6.10.2
   OS                : macOS Catalina
   Xcode             : Xcode 11.1 Build version 11A1027
```

**Comment by dwieeb** [comment]
This was likely just fixed in 0.2.9. https://github.com/ionic-team/native-run/blob/master/CHANGELOG.md#029-2019-10-15

**Comment by Incubbus** [comment]
> This was likely just fixed in 0.2.9. https://github.com/ionic-team/native-run/blob/master/CHANGELOG.md#029-2019-10-15

It was. Thanks for pointing me there.
